script.js

```diff
@@ -0,0 +1,67 @@
+document.getElementById("name").addEventListener
+        ("input", toggleCommentButton);
+document.getElementById("comment").addEventListener
```

Comment:
Ensure all elements are checked for existence before attaching event listeners. please see this code for your guide.
```
if (nameInput && commentInput && commentBtn) {
  nameInput.addEventListener("input", toggleCommentButton);
  commentInput.addEventListener("input", toggleCommentButton);
}
```